core/shared/src/main/scala/zio/Runtime.scala

```diff
@@ -174,4 +173,45 @@ object Runtime {
     val environment = r
     val platform    = platform0
   }
+
+  lazy val default = Runtime((), Platform.default)
+
+  lazy val global = Runtime((), Platform.global)
```

Comment:
With `unsafeGlobal`, it occurs to me we have 2 different meanings of `Global`: first off, global as in "powered by the global Platform", as in "powered by the global execution context". Then we have `unsafeGlobal`, which is global as in "resources that are global, for the lifetime of the application, and not deallocated until shutdown".

We need to give one of them a different name to avoid confusion.